src/web/components/Core/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { ReactNode } from 'react';
+
+import './Accordion.scss';
+
+export type AccordionProps = {
+  title: string;
+  content: ReactNode;
+  defaultExpanded: boolean;
+  isCollapsible: boolean;
+  label: ReactNode;
+};
+
+export function Accordion({
+  title,
+  content,
+  defaultExpanded,
+  isCollapsible,
+  label,
+}: AccordionProps) {
+  const itemName = 'item-1';
+  const defaultValue = defaultExpanded ? itemName : '';
+
+  return (
+    <RadixAccordion.Root
```

Comment:
I also thought about allowing multiple items, but in my testing it seems to be fine to have multiple roots